devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-routes-codestart/java/src/test/java/org/acme/MyDeclarativeRoutesTest.java

```diff
@@ -8,15 +8,16 @@
 import io.quarkus.test.junit.QuarkusTest;
 
 @QuarkusTest
-public class MyDeclarativeRoutesTest {
-	@Test
-	public void testHelloRouteEndpointWithNameParameter() {
-		given().when().get("hello-route?name=Quarkus").then().statusCode(200).body(is("Hello Quarkus !!"));
-	}
+class MyDeclarativeRoutesTest {
 
-	@Test
-	public void testHelloRouteEndpointWithoutNameParameter() {
-		given().when().get("hello-route").then().statusCode(200).body(is("Hello Reactive Route !!"));
-	}
+		@Test
+		void testHelloRouteEndpointWithNameParameter() {
+				given().when().get("hello-route?name=Quarkus").then().statusCode(200).body(is("Hello Quarkus !!"));
+		}
+
+		@Test
+		void testHelloRouteEndpointWithoutNameParameter() {
+				given().when().get("hello-route").then().statusCode(200).body(is("Hello Reactive Route !!"));
+		}
```

Comment:
I think you are using tabs here? At least, that's what GitHub seems to show? We should use 4 spaces instead.

Reply:
Fixed